doc/decisions/global_plugins.md

```diff
@@ -47,8 +51,11 @@ These hooks are not shared, so no `list` plugin is needed.
 ## Implications
 
 - remove `list` plugin
-- remove plugins that stop working, like internalnotification, or disallow global positioning for them
+- remove plugins that stop working or disallow global positioning for them
+- place `spec` very early
```

Comment:
Again, `spec` needs to the copying of metadata part directly after the storage plugins, but any additional validation must at least be done **after** all other plugins. So we either split the plugin or we mount spec twice _and_ create an API that let's plugins now the current position.

Reply:
Yes, several calls to the plugin are needed (maybe with commit function?). But I will leave this PR unchanged for now, as we should merge it for the release. A new PR&further discussion then soon after the release. I hope we can have our meeting on Tuesday but we should focus on the release first.

Reply:
I agree, let's simplify the global positions to the bare minimum we need. Adding positions can always be done without (directly) breaking other positions.

Reply:
I changed now according @kodebach's comment as I pushed a new change anyway.